src/main/java/uk/gov/hmcts/darts/common/repository/ExternalObjectDirectoryRepository.java

```diff
@@ -100,18 +100,28 @@ Optional<ExternalObjectDirectoryEntity> findMatchingExternalObjectDirectoryEntit
             WHERE eod.status in :failedStatuses
             AND eod.externalLocationType = :type
             AND eod.transferAttempts <= :transferAttempts
+            AND eod.osrUuid is null
```

Comment:
don't we have to review and make a similar update to other queries as well? for example this one:
```
    List<ExternalObjectDirectoryEntity> findByEntityAndStatus(MediaEntity media, ObjectRecordStatusEntity status);
```
This could now return a mix of DARTS and DETS EODs right?